cognite/client/_api/functions.py

```diff
@@ -394,16 +393,16 @@ def call(
         """
         identifier = IdentifierSequence.load(ids=id, external_ids=external_id).as_singleton()[0]
         id = _get_function_internal_id(self._cognite_client, identifier)
-        nonce = create_session_and_return_nonce(self._cognite_client, api_name="Functions API")
 
-        if data is None:
-            data = {}
-        url = f"/functions/{id}/call"
-        res = self._post(url, json={"data": data, "nonce": nonce})
+        with self._cognite_client.iam.sessions() as session:
```

Comment:
A similar concern here, when `wait=False`, don't you risk killing the CDF connection of the currently running function? The official docs say:

> Revoke access to a session. Revocation of a session may in some cases take up to 1 hour to take effect.

Since a function can run for up to 15 minutes (on some cloud vendors), this is not a risk I would be willing to take 😬